requirements.txt

```diff
@@ -10,7 +10,7 @@ rtree
 networkx
 bokeh<=1.4.0
 bigfeta
-opencv-contrib-python<3.4.3.0
+opencv-contrib-python
```

Comment:
I think it's fine to have it this way, but it should be mentioned that from opencv-python `3.4.3`-`3.4.12`, SIFT is not included in opencv-python-contrib.  Since SIFT's patent has expired and it is included in the regular project, the implementation in this repo now raises a DeprecrationWarning..

Reply:
were the tests failing at your end when switching back to version less than 3.4.3.0?. Some of the tests were failing for me.